lambda_code/tests/test_fhir_service.py

```diff
@@ -359,15 +408,14 @@ def test_map_disease_type_to_disease_code(self):
         nhs_number = "a-patient-id"
         disease_type = "a-disease-code"
         params = f"{self.nhsSearchParam}={nhs_number}&{self.diseaseTypeSearchParam}={disease_type}"
-        # TODO: here we are assuming disease_type=disease_code this is because the mapping is not in place yet
+        # TODO: here we are assuming disease_type=disease_code this is because the mapping is not
```

Comment:
Can this to-do be actioned as we do now have a mapping?

Reply:
I've removed the TODO and used `vaccination_procedure_snomed_codes` lookup instead

Reply:
Conflation of disease_code and disease_type - new ticket raised